README.md

```diff
@@ -1 +1 @@
-testing semantic versioning 5.0.0
```

Comment:
The code changes involve updating the semantic versioning from 5.0.0 to 5.2.0. This seems like a minor version update, so ensure that the changes made align with the version update and that the testing covers the specific changes introduced in version 5.2.0.